src/analyses/mutexAnalysis.ml

```diff
@@ -189,13 +193,21 @@ struct
 
   let return ctx exp fundec : D.t =
     begin match exp with
-      | Some exp ->
-        access_one_top ctx false false exp;
-        ctx.local
-      | None -> ctx.local
-    end
+      | Some exp -> access_one_top ctx false false exp
+      | None -> ()
+    end;
+    (* deprecated but still valid SV-COMP convention for atomic block *)
+    if String.starts_with fundec.svar.vname "__VERIFIER_atomic_" then
```

Comment:
Maybe only do this if `ana.sv-comp` is set?

Reply:
I thought about it but that makes the corresponding regression tests weird: it then requires a property file and generates witnesses files in the middle of the tests directory when running the script.

Reply:
I can make the SV-COMP option more granular to work around that somewhat, but there are still places where this can't be done conditionally. For example, `invalidate_actions` in `LibraryFunctions` could still in theory affect programs outside of whatever SV-COMP mode.

Reply:
Ah, I see. Maybe we should have one flag that controls this and the `sv-comp` includes?

If some things can not easily be done conditionally, I guess it is fine to always include them (especially given that we want to give the library mechanism an overhaul in the long run anyway).

My main reason for having a flag to control this is that it is some sort of implicit documentation that is related to `sv-comp`, making it easier to find such places (as opposed to, e.g. the ARINC stuff that just appears everywhere).